api/API_Documentation.md

```diff
@@ -307,11 +307,11 @@ Removes a book from a specified bookshelf for a user.
 ### Get List of Basic Book Data Based on Search Parameters
 
 Returns a list of books that match given search parameters and the number of books that have not yet been served to the user. The more parameters given, the narrower the search will be. If no books match the search criteria, an empty array is returned.
-* **Endpoint:** /books/search/:title/:author/:genre/:offset/:resultLength
+* **Endpoint:** /books/search
 
 * **Request Method:** GET
 
-* **URL Params:**
+* **BODY Params:**
```

Comment:
I've been looking into it and it looks like having Body params in a GET request is a breach of standards. This can lead to unforseen errors in the future. I think that GET is still the right http method to use but I think we need to find out how to pass the data in the url

Reply:
I definitely agree, its a pretty easy change to switch them back to URL params. I was just having trouble testing using url params as everything was optional + arrays became tedious to write.